src/cats/util.cljc

```diff
@@ -36,3 +42,14 @@
              IPrintWithWriter
              (-pr-writer [mv writer _]
                (-write writer (cats.protocols/-repr mv))))))
+
+(defmacro mapl [f coll]
```

Comment:
I chose `lst` here since we're working with persistent lists, not any old collection.